readme.md

```diff
@@ -22,40 +23,43 @@ $ npm install terminal-image
 const terminalImage = require('terminal-image');
 
 (async () => {
-	console.log(await terminalImage.file('unicorn.jpg'));
+	console.log(await terminalImage('unicorn.jpg'));
 })();
 ```
 
+Optionally, you can specify the `height` and/or `width` to scale the image. To
+maintain aspect ratio, define only one of them.
```

Comment:
We should always maintain aspect ratio. The image should just fit inside `width` and `height`.

Reply:
It depends of the application, but I agree it's the desired behaviour for most of the people. What about a flag enabled by default to do the fitting when both are set?

Reply:
I'm not opposed to an option for it if you can elaborate on use-cases for it?

Reply:
Alright, let's go with a flag for it: `preserveAspectRatio: true` by default.